src/themes/components/index.ts

```diff
@@ -17,7 +18,8 @@ export const components = {
   FormLabel,
   Input,
   Link,
+  Section,
   Table,
   Textarea,
-  Section,
+  VehicleReference,
```

Comment:
I doble checked with Enrique about the component's name. And this component is not a card but a vehicle reference.